ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -57,9 +57,10 @@ images:
       - registry.access.redhat.com/ubi8/ubi:latest
   to: e2e-test
 promotion:
```

Comment:
We should not be promoting this to the payload, but to a different namespace instead